tests/integration/components/myreports-list-item-test.js

```diff
@@ -14,13 +14,15 @@ module('Integration | Component | myreports list item', function(hooks) {
       title: 'Lorem Ipsum'
     });
 
-    const reportModel = await this.owner.lookup('service:store').find('report', report.id);
+    const reportModel = await this.owner
```

Comment:
This should remain a single line, multi-line is unnecessary here and makes it harder to see where the promise is that is being `await`ed.

Reply:
Ok